### PR TITLE
add moyo header

### PR DESCRIPTION
**Pull request by Yuliia-Hrinchenko1**
- [DEMO LINK](https://Yuliia-Hrinchenko1.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Yuliia-Hrinchenko1.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,70 @@
-/* Styles go here */
+html {
+  --blue-color: #00acdc;
```
Comment (vlad-demchuk) [inline src/style.css:2]: move variables to `:root` pseudo class

File: src/style.css [hunk 2]
```diff
@@ -1 +1,70 @@
...
+  list-style: none;
+  display: flex;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  /* display: flex; */
```
Comment (vlad-demchuk) [inline src/style.css:39]: remove comments

File: src/style.css [hunk 3]
```diff
@@ -1 +1,70 @@
...
+  position: relative;
+  font-family: Roboto, sans-serif;
```
Comment (vlad-demchuk) [inline src/style.css:41]: move `font-family` to `html` selector

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 4]
```diff
@@ -1 +1,72 @@
...
+}
+
+.nav__link {
+  position: relative;
+  padding: 23px 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+
```
Comment (volodymyr-soltys97) [inline src/style.css:50]: Remove the empty line for consistent formatting.

File: src/style.css [hunk 5]
```diff
@@ -1 +1,72 @@
...
+  display: flex;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+  list-style: none;
+  display: flex;
+
+}
```
Comment (volodymyr-soltys97) [inline src/style.css:37]: Remove the empty line for consistent formatting.

File: src/index.html [hunk 6]
```diff
@@ -3,12 +3,60 @@
...
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              class="nav__link"
+              data-qa="hover">
+                Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadjets" class="nav__link">Gadgets</a>
```
Comment (volodymyr-soltys97) [inline src/index.html:47]: Fix the typo in the href attribute value, change 'gadjets' to 'gadgets'.

File: src/index.html [hunk 7]
```diff
@@ -3,12 +3,60 @@
...
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo-image"
+          class="logo__image">
```
Comment (volodymyr-soltys97) [inline src/index.html:23]: Remember about code style. Check it everywhere
```suggestion
        <img
          src="images/logo.png"
          alt="logo-image"
          class="logo__image"
        >
```

File: src/index.html [hunk 8]
```diff
@@ -3,12 +3,60 @@
...
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active">Apple</a>
```
Comment (volodymyr-soltys97) [inline src/index.html:30]:
```suggestion
            <a href="#apple" class="nav__link is-active">Apple</a>
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
But let's make your code more readable

**Review by volodymyr-soltys97 — APPROVED** [review]
Awesome 🔥

**Pull request closed without merging**